translations/messages.fr.yml

```diff
@@ -764,3 +765,10 @@ delegated_access.form.access.committee: Comités
 delegated_access.created_successfully: L'accès délégué à votre espace a correctement été créé
 delegated_access.updated_successfully: L'accès délégué à votre espace a correctement été mis à jour
 delegated_access.deleted_successfully: L'accès délégué à votre espace a correctement été supprimé
+
+#
+# Senatorial Candidate
+#
+senatorial_candidate.popup.title: Charte de bonne utilisation des outils numériques
+# todo : set correct chart when available
+senatorial_candidate.popup.file_url: https://storage.googleapis.com/en-marche-fr/TECH/charte_numerique_referents.pdf
```

Comment:
I will create a copy file with name `charte_numerique_candidat_senatoriales.pdf`, could you use this file please instead? We will need just update the file in the bucket and avoid a new PR + MEP

Reply:
yes good point, I update it